.azure-pipelines/apiview-sync-staging.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+- name: 'containers'
+  type: object
+  default:
+  - originals
+  - codefiles
+
+
+stages:
+  - stage: 'Main'
+    jobs:
+      - job: 'BlobCopy'
+        pool:
+          vmImage: 'vs2017-win2016'
```

Comment:
We should be using the `windows-2019" which is the latest image.

Reply:
I will update it in my next Pr.